Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.36651&attempt_id=bffe07e3-2fff-4418-aad7-8d615907fcaa)

<details><summary>Partial log (click to expand)</summary><p>

```
./typedefs.h:104:2: error: #error Please add support for your architecture in typedefs.h
 #error Please add support for your architecture in typedefs.h
  ^~~~~
make[2]: *** [Makefile:260: audio/common/resampler/resampler.o] Error 1
make[2]: Leaving directory '/build/yate/libs/miniwebrtc'
make[1]: *** [Makefile:425: ../libs/miniwebrtc/libminiwebrtc.a] Error 2
make[1]: Leaving directory '/build/yate/modules'
make: *** [Makefile:185: modules] Error 2
builder for '/nix/store/ayqkwmba8nmk7128q3pdkkg0kncv5gn1-yate-5.4.2-1.drv' failed with exit code 2
[31;1merror:[0m build of '/nix/store/ayqkwmba8nmk7128q3pdkkg0kncv5gn1-yate-5.4.2-1.drv' failed
```
</p></details>

--- a/pkgs/applications/misc/yate/default.nix
+++ b/pkgs/applications/misc/yate/default.nix
@@ -36,7 +36,7 @@ stdenv.mkDerivation rec {
     # OpenH323 and PWlib (licensed under MPL).
     license = ["GPL" "MPL"];
     maintainers = [ lib.maintainers.marcweber ];
-    platforms = lib.platforms.linux;
+    platforms = [ "i686-linux" "x86_64-linux" ];
   };
 
 }

